deployments/helm/values.yaml

```diff
@@ -72,10 +72,10 @@ volumeMountsDiscoveryEngine:
 #resource DiscoveryEngine
 resourcesDiscoveryEngine:
   requests:
-    cpu: 50m
-    memory: 20Mi
+    cpu: 100m
+    memory: 100Mi
   limits:
-    cpu: 500m
+    cpu: 200m

```

Comment:
Lets be a bit more generous with limits. 
Please set it to
```
  limits:
    cpu: 500m
    memory: 1Gi
```
Make the same change in both places.